server/routes/induction/update/previousWorkExperienceTypesUpdateController.ts

```diff
@@ -58,14 +57,20 @@ export default class PreviousWorkExperienceTypesUpdateController extends Previou
         logger.debug(
           'Previous Work Experiences changes are only removed types so Induction can be updated without asking for work experience details',
         )
-        // call the service to update the Induction
+
+        // create an updated InductionDto with any changes to Previous Work Experiences and then map it to a CreateOrUpdateInductionDTO to call the API
+        const updatedInduction = updatedInductionDtoWithPreviousWorkExperiences(
```

Comment:
I've renamed (and refactored) the mapper method from `updatedInductionDtoWithRemovalsAndChangesToOther` to `updatedInductionDtoWithPreviousWorkExperiences`
The new name is consistent with similar mapper methods in the other Update controller classes, and it now no longer only handles removes and changes to Other .... it also handles additions 😁 , and we use it a few lines down in respect of the "mini-flow" where new work experience types have been added

Reply:
The word `previous` caught me out here. I initially thought it was work experiences that were submitted previously! 😂